### DISCUSSION TITLE
Repo filter should auto complete for starred repositories

### DISCUSSION THREAD

**Discussion by leonard84** [community/community#12802] [Other Feature Feedback, Questions, & Ideas]
Currently, the `repo:` filter will only start auto-completion, once the user has has finished typing in the organization part of the query, e.g. `repo:spockframework/`. It would be really helpful, if it would support auto completion of organizations where users have a starred repository, or where they are a member of. This would be a good compromise between a full auto completion across all organizations/users and having none.

**Comment by colinwm** [comment]
This is a great idea, I'll look into implementing it!
